pkg/utils/netlink_manager.go

```diff
@@ -0,0 +1,96 @@
+package utils
```

Comment:
In the root folder:
1) make
2) go mod vendor
3) docker run -v "$PWD":/src -w /src vektra/mockery --recursive=true --name=NetlinkManager --output=./pkg/sriov/mocks/ --filename=netlink_manager_mock.go
4) docker run -v "$PWD":/src -w /src vektra/mockery --recursive=true --name=pciUtils --output=./pkg/sriov/mocks/ --filename=pci_utils_mock.go --exported

I updated the files with the autogenerated files from mockery. I ran "sudo make test" and it seems everything is working properly.

Reply:
perhaps some doc put under the "mocks" dir  ?

Reply:
afaik, its a common practice to put mocked interfaces of a package under `mocks` folder in the same location of the package itself.

this can be addressed as a followup by creating a `Developer.md` with a section describing this.
imo it can be toplevel doc next to `README.md` WDYT ?

Reply:
I think its a good idea Adrian and Eoghan. I put this document "Developer.md" at the root of the project directory.